lib/plugins/boluswizardpreview.js

```diff
@@ -205,7 +225,20 @@ function pushInfo(prop, info, sbx) {
       var carbEquivalent = Math.ceil(Math.abs(sbx.data.profile.getCarbRatio() * prop.bolusEstimateDisplay));
       info.unshift({label: 'Carb Equivalent', value: prop.bolusEstimateDisplay + 'U * ' + sbx.data.profile.getCarbRatio() + ' = ' + carbEquivalent + 'g'});
       info.unshift({label: 'Current Carb Ratio', value: '1U / ' + sbx.data.profile.getCarbRatio() + ' g'});
-      info.unshift({label: '-BWP', value: prop.bolusEstimateDisplay + 'U, maybe covered by carbs?'});
+      
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Expected '{' and instead saw 'info'.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/944034239/issues/source?bid=2452379&fileBranchId=2847953#l229)